docs/docs/30-administration/10-server-config.md

```diff
@@ -20,9 +20,9 @@ services:
 +     - WOODPECKER_OPEN=true
 ```
 
-You can **also restrict** registration, by keep registration closed and ...  
-... **adding** new **users manually** via the CLI: `woodpecker-cli user add`, or  
-... allowing specific **admin users** via the `WOODPECKER_ADMIN` setting, or  
+You can **also restrict** registration, by keep registration closed and ...
```

Comment:
removing newlines intentional?

Reply:
It was done by prettier. If its not a double new-line its not considered anyways